pkgs/development/compilers/rust/bootstrap.nix

```diff
@@ -3,16 +3,16 @@
 let
   # Note: the version MUST be one version prior to the version we're
   # building
-  version = "1.29.2";
+  version = "1.30.0";
```

Comment:
@GrahamcOfBorg one major version? You said people from `#rust` told you said.

Reply:
I don't know either, but I think we've been doing minor releases.